tests/OptionalTests.cs

```diff
@@ -73,6 +73,14 @@ where Fail<bool>()
             Assert.That(result, Is.EqualTo(none));
         }
 
+        [Test]
+        public void SelectManySimple() //TODO rename?
+        {
+            var result = Option.Some(42).SelectMany(n => Option.Some(new string((char) n, n)));
```

Comment:
For consistency with the rest of the tests, this should be also expressed in query syntax, as in:

```c#
var result =
    from n in Option.Some(42)
    from s in Option.Some(new string((char) n, n))
    select s;
```

Reply:
Wasn't sure of the difference with the other overload ^^